src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,880 @@
+
+	private JTextField cdpTextField;
+
+	private JLabel messageLabel;
+
+	private JCheckBox checkBoxSubmitYesSamples;
+
+	private JCheckBox checkBoxSubmitFailingSamples;
+
+	private QuickSubmitter submitter = new QuickSubmitter();
+
+	private JTextArea textArea;
+
+	private SimpleDateFormat hhMMSSformatter = new SimpleDateFormat("hh:mm:ss");
+
+	private int linenumber = 0;
+	/**
+	 * @wbp.nonvisual location=162,159
+	 */
+	private final ButtonGroup judgingTypesButtonGroup = new ButtonGroup();
+	
+	private final ButtonGroup problemsButtonGroup = new ButtonGroup();
+    
+	private final ButtonGroup languagesButtonGroup = new ButtonGroup();
+    
+	
+	/**
+	 * List of selected solutions names and dirs.
+	 */
+	private SubmissionSolutionList submissionSolutionList = null;
+	
+	/**
+	 * List of selected Problems.
+	 * 
+	 */
+	private List<Problem> selectedProblemList = null;
+	
+    private List<Language> selectedLanguageList = null;
+    
+	private int[] selectedJudgingTypeIndexes;
+	
+	private int[] selectedProblemsIndexes;
+	
+	private int[] selectedLanguagesIndexes;
+	
+	private JLabel selectedNosLabel = new JLabel("None selected");
+	
+	private JRadioButton submitAllJudgingTypesRadioButton = new JRadioButton("Submit All");
+	
+	private JRadioButton submitSelectedJudgingTypeRadioButton = new JRadioButton("Submit Selected");
+	
+    private JCheckBox checkBoxProblems = null;
+
+    private JRadioButton selecteAllProblemsRadioButton = null;
+    private JRadioButton submitAllProblemsRadioButton;
+
+    private JRadioButton submitSelectedProblems = null;
+    private JRadioButton submitSelectedProblemsRadioButton;
+
+    private JLabel selectedProblemsLabel = null;
+
+    private JButton selectProblemsButton = null;
+
+    private JCheckBox checkBoxLanguage = null;
+
+    private JRadioButton selecteAllLanguages = null;
+    private JRadioButton submitAllLanguagesRadioButton;
+
+    private JRadioButton submitSelectedLanguages = null;
+    private JRadioButton submitSelectedLanguagesRadioButton;
+
+    private JLabel selectedLanguagesLabel = null;
+
+    private JButton selectLanguageButton = null;
+
+
+    private Log log;
+
+    private boolean debug22Flag = false;
+
+    // TODO On Admin update of Languages or Update of Problems - clear selected index arrays
+
+	public SubmitSampleRunsPane() {
+		super();
+		setLayout(new BorderLayout(0, 0));
+
+		JPanel centerPane = new JPanel();
+		centerPane.setLayout(new BorderLayout());
+		add(centerPane, BorderLayout.CENTER);
+
+		JSplitPane splitPane = new JSplitPane();
+		splitPane.setOrientation(JSplitPane.VERTICAL_SPLIT);
+		centerPane.add(splitPane, BorderLayout.CENTER);
+
+		JPanel controlsPane = new JPanel();
+		controlsPane.setPreferredSize(new Dimension(400, 400));
+
+		controlsPane.setLayout(null);
+
+		JLabel cdpConfigDirLabel = new JLabel("CDP config dir");
+		cdpConfigDirLabel.setToolTipText("CDP Location for sample source files");
```

Comment:
The use of the term "sample" in this tooltip has the potential to generate confusion -- it could easily be misconstrued as having something to do with "sample test data cases".  I think it should say (something like) "CDP location for judge's submission files").